server/src/main/java/org/jboss/as/server/BootstrapListener.java

```diff
@@ -92,11 +101,36 @@ protected void done(final long bootstrapTime, final StabilityStatistics statisti
         final int started = statistics.getStartedCount();
         if (failed == 0 && problem == 0) {
             ServerLogger.AS_ROOT_LOGGER.startedClean(prettyVersion, bootstrapTime, started, active + passive + onDemand + never + lazy, onDemand + passive + lazy);
+            createStartupMarker("success", startTime);
         } else {
             ServerLogger.AS_ROOT_LOGGER.startedWitErrors(prettyVersion, bootstrapTime, started, active + passive + onDemand + never + lazy, failed + problem, onDemand + passive + lazy);
+            createStartupMarker("error", startTime);
         }
     }
 
+    private void createStartupMarker(String result, long startTime) {
+        String tmpDir = isStandalone ? System.getProperty("jboss.server.temp.dir") : System.getProperty("jboss.domain.temp.dir");
```

Comment:
No system properties. See my comments above about where to find the location of these dirs.